app/controllers/idv/phone_controller.rb

```diff
@@ -199,16 +199,16 @@ def async_state_done(async_state)
       end
     end
 
-    def is_req_from_frontend?
+    def req_from_frontend?
```

Comment:
Could be a good opportunity to unabbreviate this for clarity.

```suggestion
    def request_from_frontend?
```

Or

```suggestion
    def frontend_request?
```

Reply:
Good idea! Went with `frontend_request?` and followed up in 4aade4d18bdc2bea417e01b553173bcbe53b4133